{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/news/models.py

```diff
@@ -227,6 +234,9 @@ def render_card(self):
             'article': self,
         })
 
+    def article_length(self):
```

Comment:
This might be better as a `cached_property` as it doesn't change over the lifetime of the instance.

Reply:
(Also, `word_count` might be a better name for this function.)